component/client/core/ClientCore.tsx

```diff
@@ -24,7 +24,7 @@ class ClientCore extends React.Component<any, ClientCoreState> {
 
     // Only change initialize status if setting is found.
     storage.get(STORAGE_KEY_INITIALIZED).then((hasInitalized) => {
-      if (hasInitalized) this.setState({initialized: hasInitalized});
+      if (hasInitalized) this.setState({initialized: hasInitalized === "true"});
```

Comment:
`true === "true"` this returns false. 
I do not store strings when setting this variable. It will and should come back from chrome storage as a boolean. You can add `hasInitialized: boolean` if needed.